client/src/components/pages/boilers/Boilers.jsx

```diff
@@ -2,9 +2,13 @@ import React, { useState } from 'react';
 import boilers from './Boilers.json';
 import BoilersTable from './BoilersTable';
 import AddBoiler from './AddBoiler';
-import EditBoiler from './EditBoiler';
+import './Boilers.css';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
 
-const Boilers = () => {
+import { postBoiler as postBoilerB } from '../../../redux/actions/Boilers.actions';
```

Comment:
PostBoilerB is not descriptive. If you would want it to have an alias name it should be something like PostBoilerAction